simulation.py

```diff
@@ -0,0 +1,73 @@
+"""
+A function to simulate alpha
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import TRADING_CONFIG, DATA_LIST_ALL
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = raw_data['workdays']
+    invest_data = {index : value
+                   for index, value in raw_data.items() if index != 'workdays'}
```

Comment:
If we don't really need `raw_data` later, we can just remove `workdays` from `raw_data` instead of creating a new equivalent object.